pages/apis/webhooks/integrations.md.erb

```diff
@@ -4,13 +4,12 @@ There are a number of third party services you can use with Buildkite webhooks.
 
 <%= toc %>
 
-## AWS SNS with AWS Lambda
+## AWS Lambda
 
-[AWS SNS](https://aws.amazon.com/sns/) is an Amazon pub-sub service providing messaging and mobile notifications. [AWS Lambda](https://aws.amazon.com/lambda/) is an Amazon service for running code functions in response to events like SNS messages or HTTP requests.
+[AWS Lambda](https://aws.amazon.com/lambda/) is an AWS service for running code functions in response to events. [AWS API Gateway](https://aws.amazon.com/api-gateway/) is a service for creating HTTP endpoints for your Lambda functions, and [AWS SNS](https://aws.amazon.com/sns/) is a messaging service which can be used to trigger many Lambda functions from webhook events. These three services can be combined in a variety of ways to consume webhooks.
```

Comment:
```suggestion
[AWS Lambda](https://aws.amazon.com/lambda/) is a service for running functions, and when combined with [AWS API Gateway](https://aws.amazon.com/api-gateway/), can be used to process your Buildkite webhooks.
```

Reply:
The wording here is awkward. The idea is that you need Lambda for the code, and then API Gateway to create a HTTPS endpoint you use as your Buildkite webhook endpoint. SNS is unncessary (but cool, and potentially useful) so I've left that to the examples below.